CMakeLists.txt

```diff
@@ -66,37 +66,79 @@ message(STATUS "[i] iceoryx2 dev setup Rust feature flags (only used when 'RUST_
 
 add_rust_feature(
     NAME IOX2_FEATURE_STD
-    DESCRIPTION "Build iceoryx2 with std library."
+    DESCRIPTION "Build iceoryx2 with std library"
     DEFAULT_VALUE ON
-    RUST_FEATURE "std"
+    RUST_FEATURE "iceoryx2-ffi-c/std"
```

Comment:
@elBoberido I changed this to use the feature flag defined on the package being built. Let me know if this is not correct.